### DISCUSSION TITLE
save some settings

### DISCUSSION THREAD

**Discussion by kalle07** [lllyasviel/Fooocus#2191] [Ideas]
i have insttaled the newest windows ZIP file

have i over read some , or ?!?!

where i can save some settings

whyi must every time i load foocus adjust all my settings !??   : D

**Comment by eddyizm** [comment]
You can save some settings as a preset and then launch that preset to get back to where you want to be.

**Comment by kalle07** [comment]
isee 
ok, tell me how, like iam a newbee ^^
save dafaultmodel
save dont take any styles
image numer "one"
...

**Comment by SGThirkell** [comment]
You change the config.txt file in the Fooocus folder
For example for image number you would add the line
"default_image_number": 1,

There are examples in the config_modification_tutorial.txt file

**Comment by kalle07** [comment]
THX
i see its only for base python programmers

[[[{{}}]]  ?!?

may you can add a save preset button   : D
